Card/src/main/java/com/paypal/android/card/CardAPI.kt

```diff
@@ -12,17 +11,28 @@ internal class CardAPI(
     private val api: API,
     private val requestFactory: CardAPIRequestFactory = CardAPIRequestFactory()
 ) {
-    suspend fun confirmPaymentSource(orderID: String, card: Card): ConfirmPaymentSourceResult {
+    suspend fun confirmPaymentSource(orderID: String, card: Card): CardResult {
         val apiRequest = requestFactory.createConfirmPaymentSourceRequest(orderID, card)
         val httpResponse = api.send(apiRequest)
+        val correlationId = httpResponse.headers["Paypal-Debug-Id"]
         return if (httpResponse.status == HTTP_OK) {
             runCatching {
                 val json = PaymentsJSON(httpResponse.body)
                 val status = json.getString("status")
                 val id = json.getString("id")
-                ConfirmPaymentSourceResult(response = OrderData(id, OrderStatus.valueOf(status)))
+
+                CardResult.Success(
+                    orderID = id,
+                    status = OrderStatus.valueOf(status)
+                )
             }.recover {
-                ConfirmPaymentSourceResult(error = OrderError("PARSING_ERROR", "Error parsing json response."))
+                CardResult.Error(
+                    orderError = OrderError(
+                        "PARSING_ERROR",
+                        "Error parsing json response."
```

Comment:
Just out of curiosity, do we follow a specific standard for arguments naming? Since there are three expected and only two are provided, should the namings be included?